src/base/rss/rssfolder.cpp

```diff
@@ -189,7 +189,6 @@ QHash<QString, FeedPtr> Folder::getAllFeedsAsHash() const
 
 bool Folder::addFile(const FilePtr &item)
 {
-    Q_ASSERT(!m_children.contains(item->id()));
```

Comment:
Consistent in what way? Looking at all usages of the function, I can at least say that none of them actually make use of the returned boolean, they all just blindly add. 

While looking at the usages though, I did find a couple places in rss_imp I missed updating when I created the addFeed helper. So fixing those may also fix the cases that were triggering the assertion. Even so though I still believe it would be better off removed.